dtfs-central-api/src/v1/controllers/portal/facility/get-facilities.controller.js

```diff
@@ -14,7 +14,8 @@ exports.findAll = findAll;
 
 const findAllFacilitiesByDealId = async (dealId) => {
   const collection = await db.getCollection('facilities');
-  const facilities = await collection.find({ dealId }).toArray();
+  // BSS facilities
+  const facilities = await collection.find({ dealId, $or: [{ facilityType: 'Bond' }, { facilityType: 'Loan' }] }).toArray();
```

Comment:
Should this function also return GEF facilities?

Reply:
that call is only for BSS facilities -  and only `Bond` & `Loan` are part of bss facilities

Reply:
Should we consider renaming the function if it is for BSS onlye `findAllBssFacilitesByDealId`?

Reply:
I'm going to do that, yes. but not in this PR because there will be a few things moving and I wanted to test it separately.
I was planning to combine some of the files together as well in central-api since now we have both deals and facilities in the same collection and there is no need to have separate folders/files for bss/gef